src/styles/components/pages/home/Module/OfferTile/_OfferTile.scss

```
@@ -9,12 +9,15 @@
 
   .otw-venue {
     @include caption();
+    overflow: hidden;
+    text-overflow:ellipsis;
   }
 
   .otw-offer-info {
     @include caption();
-
     color: $grey-dark;
+    overflow: hidden;
+    text-overflow:ellipsis;
```

```suggestion
    text-overflow: ellipsis;
```
Faut que tu regardes ta configuration IDE car avec les nouvelles règles que j'ai mis en place, il devrait te le dire.

---

Merci , c'est fait